### DISCUSSION TITLE
read/write config with command line

### DISCUSSION THREAD

**Discussion by bytelinker** [pvvx/ATC_MiThermometer#578] [Q&A]
Hello, 

the configuration can be changed with TelinkMiFlasher.html. Is there a possibility to read and write config with command line or code, please? I need to change it in many sensors. 

Thank you very much.

**Comment by pvvx** [comment]
https://github.com/pvvx/ATC_MiThermometer/tree/master/python-interface

**Comment by bytelinker** [comment]
Telink Flasher is offering to change

Temperature Slope factor:  175.00  , Zero offset: -45.00
Humidity Slope factor: 100.00 , Zero offset: 0.00

both slope and offset for temp and humidity. I'didn't find those settings in the python interface for batch commandline usage. Are they avaliable?
